#Posição

---

#para onde ir

projeto_semear/Pose to_go
uint32 container_escolhido  # 0=esquerda, 1 = direita, 2 = só tem vermelho aqui
uint32 cor                  # 13=azul, 12=verde, 14=vermelho //mudar aqui para a .msg
uint32 pilha                # Posição da pilha que pegou